internal/controller/rsm/plan_builder.go

```diff
@@ -56,26 +56,26 @@ func init() {
 
 // PlanBuilder implementation
 
-func (b *csSetPlanBuilder) Init() error {
-	csSet := &workloads.ConsensusSet{}
-	if err := b.cli.Get(b.transCtx.Context, b.req.NamespacedName, csSet); err != nil {
+func (b *rsmPlanBuilder) Init() error {
```

Comment:
a private struct with public receiver functions seems odd to me.

Reply:
done: [ed95314](https://github.com/apecloud/kubeblocks/pull/3733/commits/ed9531499b9264b9d310314233839f85bc09ba6b)